src/App.js

```diff
@@ -62,20 +62,24 @@ const App = () => {
         <ion-icon name="search-sharp"></ion-icon>
       </div>
       <Modal
-        className="bg-primary color-secondary max-w-lg mx-auto my-32 p-8"
+        className="bg-primary border-2 color-secondary max-w-lg mx-auto my-32 p-8"
```

Comment:
Changing `p-8` to `p-6` gives the modal window more room to display the content, giving the description "After logging in with Freesound, you'll be sent back here!" enough room to all fit on one line, again, like it does in the [Figma](https://www.figma.com/file/jeyLWmyn78DjqSmbWcJOEd/Tide?node-id=798%3A82)

![modal window with [padding changed from 2rem to 1.5rem](https://tailwindcss.com/docs/padding#add-padding-to-all-sides)](https://user-images.githubusercontent.com/43425812/97794435-3bbae000-1bb7-11eb-92ab-7f94c68df845.png)

In case anyone needs to know for future reference, I updated the horizontal padding on the `Send me there!` button from `px-4` to `px-6` and the horizontal padding on the `Maybe Later` button from `px-4` to `px-8` in the image just uploaded.

Still need to fix 1) background not becoming "transparent" like it was before I had to override the overlay's default styles to update the overlay's `z-index`, 2) make the border only appear when the night theme is active, and 3) "lock" the `X` symbol to close the window to the upper-righthand corner of the modal window.